react/AdminExample.tsx

```diff
@@ -1,12 +1,25 @@
-import React, { Component } from 'react'
-import { FormattedMessage } from 'react-intl'
+import React, { Component } from "react";
+import { FormattedMessage } from "react-intl";
+import { Layout, PageBlock, PageHeader } from "vtex.styleguide";
 
 class AdminExample extends Component {
   public render() {
     return (
-      <FormattedMessage id="admin-example.hello-world" />
-    )
+      <div className="h-100">
```

Comment:
Should be useless with Styleguide `8.x`: https://kevin--ambienteqa.myvtex.com/admin/customercredit/config `<Layout` is directly used: https://github.com/vtex/admin-customer-credit/blob/feature/state-query-view-pattern/react/pages/configuration/view.js#L51

Reply:
this doesn't even work 😢

Reply:
I made a fix in the next commit, but this should not be the final solution :'(